tests/test_github_automatic_fields.py

```diff
@@ -0,0 +1,72 @@
+"""
+Test that with no fields selected for a stream automatic fields are still replicated
+"""
+from tap_tester import runner, connections
+
+from base import TestGithubBase
+
+
+class TestGithubAutomaticFields(TestGithubBase):
+    """Test that with no fields selected for a stream automatic fields are still replicated"""
+
+    @staticmethod
+    def name():
+        return "tap_tester_github_automatic_fields"
+
+    def test_run(self):
+        """
+        - Verify that for each stream you can get multiple pages of data
+            when no fields are selected.
+        - Verify that only the automatic fields are sent to the target.
+        - Verify that all replicated records have unique primary key values.
+        """
+
+        # BUG TDL-16137 `team_memberships` stream is not passing run_and_verify_sync()
+        expected_streams = self.expected_check_streams()
```

Comment:
This method is skipping more than just the `team_memberships` stream.

Reply:
using `expected_streams` now that bug has been identified